packages/api/drizzle/0008_add_secrets.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE `secrets_to_sessions` (
+	`id` integer PRIMARY KEY NOT NULL,
+	`session_id` text NOT NULL,
+	`secret_id` integer NOT NULL,
+	FOREIGN KEY (`secret_id`) REFERENCES `secrets`(`id`) ON UPDATE no action ON DELETE no action
```

Comment:
This needs a on delete cascade otherwise we cannot delete the secret on the secrets page without an exception (which currently crashes the server)

Reply:
Good catch! Should be fixed.